Bag of words/index.html

```diff
@@ -4,17 +4,14 @@
   <meta charset="utf-8" />
   <title>SuperStore Sales Dashboard</title>
   <link rel="stylesheet" href="ui/bootstraptheme.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
 </head>
-<body>
-  <div class="formhandler" data-src="data"></div>
-  <form action="Query">
-    <input type="text" name="query" placeholder="Input Query" required> <br>
-    <input type="submit" value="Submit">
-  </form>
-  <div id="chart"></div>
-  <div class="container1"></div>
-
-</body>
+<frame>
+   <input id="Searchbox" type="text" name="query" placeholder="Input Query" required > <br>
+   <button>Submit</button>
+   <div id="box"></div>
```

Comment:
Try to make this work by pressing Enter, instead of clicking the Submit button. [Optional]